addon/components/relationship-select.hbs

```diff
@@ -0,0 +1,25 @@
+<input
+  class="hidden-input"
+  value={{or (or @selected.id @selected.length) ""}}
+  ...attributes
+  {{did-insert (perform this.fetchModels)}}
+/>
+{{#with
+  (component
+    (if @multiple "power-select-multiple" "power-select")
+  ) as |RelationPowerSelect|
+}}
+  <RelationPowerSelect
+    @searchEnabled={{true}}
```

Comment:
I'd suggest only enabling the search if you have more than e.g 5 options

Reply:
Done :+1: